package.json

```diff
@@ -105,6 +108,7 @@
     "watchify": "^3.7.0"
   },
   "dependencies": {
+    "@babel/plugin-transform-spread": "^7.2.2",
```

Comment:
I think maybe this should be a dev dependency like the other babel dependencies? You can save as a dev dependency using `yarn add @babel/plugin-transform-spread --dev`

Reply:
Agreed, I saw where that landed, but this doesn't actually fix my problem.  I forgot to relink when I was testing, so I wasn't testing what I thought.